src/lib/optimizer/abstract_syntax_tree/abstract_ast_node.hpp

```diff
@@ -123,6 +123,9 @@ class AbstractASTNode : public std::enable_shared_from_this<AbstractASTNode> {
       const std::shared_ptr<AbstractASTNode>& right_child = nullptr) const;
   // @}
 
+  /**
+   * @returns the names of the columns this node outputs without any alias added by this node
+   */
```

Comment:
Is this comment added for clarification or did you change the semantics of this function? Or: is this still true for all other nodes?

Reply:
just docs